.gitignore

```diff
@@ -1,3 +1 @@
 node_modules
-/client/live.bundle.js
-/client/index.bundle.js
```

Comment:
This change wasn't in the original pull request. I included live.bundle.js and index.bundle.js in my branch so that I could npm install from my repository. The added files and the change to .gitignore can be reverted (the changes in f170088)